tools/bin/hawq_ctl

```diff
@@ -737,25 +739,28 @@ class HawqStart:
             logger.info("Segments started successfully")
         return node_init.return_flag
 
-    def _start_rps(self):
-        logger.info("Start ranger plugin service")
+    def _start_rps(self, master_or_standby):
```

Comment:
It's better not identify parameter as "master_or_standby", since this looks like a boolean value, and is not valid for internal implementation of start_rps function. I'd recommend to rename it to "hostname" or other similar name.